slo-libsonnet/latency-burn.libsonnet

```diff
@@ -1,12 +1,13 @@
 local util = import '_util.libsonnet';
 {
-  burn(param):: {
+  latencyburn(param):: {
     local slo = {
       metric: error 'must set metric for latency burn',
       selectors: error 'must set selectors for latency burn',
       latencyTarget: error 'must set latencyTarget latency burn',
       latencyBudget: error 'must set latencyBudget latency burn',
       labels: [],
+      codeSelector: 'code',
```

Comment:
I was also thinking about this and maybe we can make it even more generic by using something like `excludeSelector: ['code!~"5.."']`, but AFAIC this is not possible with a regex match right?

Reply:
This should actually be possible then, yes. Cause then we simply need the string and concatenate. Checking...

Reply:
We discussed this outside of GitHub. It would be nice to have this at some point but needs more attention...